lib/vehicle.rb

```diff
@@ -1,18 +1,20 @@
 require 'date'
 
 class Vehicle
-  attr_reader :vin,
-              :year,
-              :make,
-              :model,
-              :engine
+  attr_accessor :vin, :year, :make, :model, :engine, :registration_date, :plate_type
 
```

Comment:
similarly here.. consider which of these attributes can be readers, and which of them should be accessors.

Reply:
It looks like you change `plate_type` and `registration_date` in your Facility class, so those will probably be the only accessor's that you need to list, and the rest can be readers.